src/components/app/AppStatus.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export default function AppStatus({ appStatus }: { appStatus: string }) {
+    const color = appStatus?.toLocaleLowerCase() === 'notdeployed' ? 'not-deployed' : appStatus?.toLocaleLowerCase()
```

Comment:
Do we need to put an optional check here?

Reply:
Instead can we put the check at where we're using `AppStatus`?

Reply:
done